src/thx/Arrays.hx

```diff
@@ -205,7 +205,7 @@ comparison is ==.
     var result = [];
 
     if (array.length <= 1)
-      return result;
```

Comment:
Yeah, I can change it to that... wasn't sure what was the best way to change a ReadonlyArray to an Array